recipes/carve/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "carve" %}
+{% set version = "2.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ngodber/carve/archive/v{{ version }}.tar.gz
+  sha256: 20481918af488fc92694bf1d5bdd6351ad73a0b64fbe4373e1f829a7b0eeff63
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage("carve", max_pin="x.x") }}
+
+requirements:
+  build:
+    - cmake
+    - boost  
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  run:
+    - boost  
```

You don't seem to need Boost.Python, so use only the C++ parts of boost.
```suggestion
  host:
    - boost-cpp
  run:
    - boost-cpp  
```